add subscriber feature for event bus (#349)
patchlevel/event-sourcing: docs/pages/event_bus.md

### Default event bus

> The library also delivers a light-weight event bus. This can only register listener/subscriber and dispatch events.

Reviewer: ```suggestion
The library also delivers a light-weight event bus for which you can register listeners/subscribers and dispatch events.
```

> The library also delivers a light-weight event bus. This can only register listener/subscriber and dispatch events.

Reviewer: ```suggestion
The library also delivers a light-weight event bus for which you can register listeners/subscribers and dispatch events.
```

!!! warning

    If you only want to listen to certain messages, 
    then you have to check it in the `__invoke` method or use the subscriber.

## Subscriber

A `Subscriber` is a listener, except that it has implemented the invoke method itself. 
Instead, you can define your own and multiple methods and listen for specific events with the attribute `Handle`.

```php
use Patchlevel\EventSourcing\Attribute\Handle;
use Patchlevel\EventSourcing\EventBus\Listener;
use Patchlevel\EventSourcing\EventBus\Message;

> $listener = new class extends Subscriber 

Reviewer: ```suggestion
final class WelcomeSubscriber extends Subscriber 
```

I think this reduces the complexity of the example, since a normal class is more convenient to understand then an anonymous class.
